Tests/Acceptance/MasterpassCheckoutTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+namespace Wirecard\Oxid\Tests\Acceptance;
+
+use Wirecard\Oxid\Model\MasterpassPaymentMethod;
+
+/**
+ * Acceptance tests for the Masterpass checkout flow.
+ */
+class MasterpassCheckoutTest extends CheckoutTestCase
+{
+    public function getPaymentMethodName()
+    {
+        return MasterpassPaymentMethod::getName(true);
+    }
+
+    public function testCheckout()
+    {
+        $this->goThroughCheckout();
+        $this->goThroughExternalFlow();
```

Comment:
If you implement the `assertRedirected` method above, if should be enough to call it here.
```suggestion
        $this->assertRedirected();
```